src/Cache.js

```diff
@@ -1,55 +1,38 @@
 'use strict'
 
-const fs   = require('fs')
-const path = require('path')
-
 let filePath
 let cache = {}
 
-class Cache {
-  static set(key, value) {
-    return new Promise((resolve, reject) => {
-      cache[key] = value
-      if(filePath) {
-        // console.log("write cache:", filePath, JSON.stringify(cache, null, 2))
-        fs.writeFile(filePath, JSON.stringify(cache, null, 2) + "\n", resolve)
-      } else {
-        resolve()
-      }
-    })
-  }
+let store = typeof window !== 'undefined' && window.localStorage
+
+if (!store) {
+  const LocalStorage = require('node-localstorage').LocalStorage
+  store = new LocalStorage('./scratch')
+}
```

Comment:
Re-use `cacheFile` param I guess. loadCache(cacheFile) should be the full path of the file. from there, you can separate the file and path from `cacheFile`. loadCache should be only called once before cache is ready to use (ie. don't have to pass the cache file path to get/set) . Does that make sense?